### PR TITLE
SIG M&G - learning from SD

### PR DESCRIPTION
**Pull request by geekygirldawn**
Updated the SIG M&G responsibilities with a few new things based on what we learned in San Diego.

@detiber - what do you think? Is there anything you can think to add?

/sig contributor-experience
/area contributor-summit

### CODE REVIEW

**Comment by mrbobbytables** [comment]
Much appreciated :D 
/approve

File: events/events-team/content/sig-contrib-events.md [hunk 1]
```diff
@@ -38,14 +38,16 @@ Time Commitment
 - Expect to spend a fair amount of time following up with SIGs and WGs to get the name of their representatives.
 - Add these representatives to a calendar notice for the SIG Meet and Greet.
 - Work with the registration lead to make sure that representatives are registered for the summit and follow up with any representatives who have not registered.
-- Make sure that we have table stands and buttons for each SIG that are organized and easy to hand out at the beginning of the event.
+- Make sure that we have table stands and buttons for each SIG that are organized and easy to hand out at the beginning of the event. These were done in alphabetized reusable bags for San Diego, which worked really well. 
+- Create a plan for organizing SIGs / WGs at tables to make it easier for attendees to find SIGs (alphabetical would be a logical choice). Make sure that this plan is communicated to volunteers and other organizers.
```
Comment (detiber) [inline events/events-team/content/sig-contrib-events.md]: One nit here is that there are SIGs and WGs that are closely related that are not necessarily close alphabetically. As some representatives might be representing multiple groups, we may end up with odd exceptions if we try to do alphabetical order.

I'm wondering if we can improve visibility of where SIGs and WGs are in the room through better signage and table/seating layout instead.

Comment (geekygirldawn) [reply events/events-team/content/sig-contrib-events.md]: Excellent suggestion, thank you!

**Pull request merged**